### PR TITLE
Add Changelog to project_urls

### PR DESCRIPTION
**Pull request by adamchainz**
Make it easy to jump from the PyPI page to what has change.d

### Feature or Bugfix
- Feature

### Purpose
description says it all

### Detail
n/a

### Relates
n/a

### CODE REVIEW

**Comment by AA-Turner** [comment]
What does this render as in PyPI? Does it have a special icon?

A

File: setup.py [hunk 1]
```diff
@@ -67,6 +67,7 @@
     long_description_content_type='text/x-rst',
     project_urls={
         "Code": "https://github.com/sphinx-doc/sphinx",
+        "Changelog": "https://www.sphinx-doc.org/en/master/changes.html",
```
Comment (AA-Turner) [inline setup.py:70]: For consistency with the name of the file.
```suggestion
        "Changes": "https://www.sphinx-doc.org/en/master/changes.html",
```

**Comment by adamchainz** [comment]
Yes it has an icon: https://github.com/pypa/warehouse/blob/3c93c0365ea7afafbbe0bbe5e76f9a6c29559afd/warehouse/templates/packaging/detail.html#L24

See example here: https://pypi.org/project/apig-wsgi/

File: setup.py (hunk 1 above)
Comment (adamchainz) [reply setup.py:70]: I would stick with "changelog" to be consistent with the page title and the many other projects that have a "changelog" link in their sidebar (I must have contributed this to like a hundred repos now).

**Review by AA-Turner — APPROVED** [review]

**Review by tk0miya — APPROVED** [review]
LGTM

**Comment by tk0miya** [comment]
Thank you for your contribution. Please wait for the next release, 5.1.0.

**Comment by adamchainz** [comment]
Thank you for maintaining Sphinx 😃

**Pull request merged**